core/src/execution/datafusion/expressions/scalar_funcs.rs

```diff
@@ -130,6 +133,10 @@ pub fn create_comet_physical_fun(
             let func = Arc::new(spark_xxhash64);
             make_comet_scalar_udf!("xxhash64", func, without data_type)
         }
+        "chr" => {
```

Comment:
Is the expected behavior returning null if the input is null?
Another approach could be converting `chr(child)` to `If(IsNull(child), literalNull, chr(child))` to reuse existing `chr`

Reply:
Question - 
 IsNull(child) won't work because we have an element there - 0 (int). Hence we will hit chr(Child) always. Is my understanding correct?

Reply:
@kazuyukitanimura The null handling was already correct. The issue was specifically for the input value zero.

Reply:
I suppose it would be possible to build an `If(child == lit(0), lit("0"), chr(child))` but seems like that would add more runtime overhead and complexity

Reply:
Got it, makes sense